src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid_item.tsx

```diff
@@ -66,11 +68,24 @@ const Item = React.forwardRef<HTMLDivElement, Props>(
       printViewport__vis: container.getInput().viewMode === ViewMode.PRINT,
     });
 
+    useEffect(() => {
+      let panelElement;
```

Comment:
Caught this when playing with the scrolling behaviour - looks like this `panelElement` variable isn't actually used

Reply:
I just saw that too haha. I'll remove it